messaging/src/main/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessor.java

```diff
@@ -715,6 +725,29 @@ public Builder clock(@Nonnull Clock clock) {
             return this;
         }
 
+        /**
+         * Enables the {@link Coordinator} to {@link WorkPackage#extendClaim() extend the claims} of its
+         * {@link WorkPackage WorkPackages}.
+         * <p>
+         * Enabling "coordinator claim extension" is an optimization as it relieves this effort from the
+         * {@code WorkPackage}. Toggling this feature may be particularly useful whenever the event handling task of the
+         * {@code WorkPackage} is lengthy. Either because of a hefty event handling component or because of a large
+         * {@link PooledStreamingEventProcessor.Builder#batchSize(int)}.
+         * <p>
+         * In both scenarios, there's a window of opportunity that the {@code WorkPackage} is not fast enough in
+         * extending the claim itself. Not being able to do so potentially causes token stealing by other instances of
+         * this {@link PooledStreamingEventProcessor}, thus overburdening the overall event processing task.
+         * <p>
+         * Note that enabling this feature will result in more frequent invocation of the {@link TokenStore} to update
+         * the tokens.
+         *
+         * @return The current Builder instance, for fluent interfacing.
+         */
+        public Builder enabledCoordinatorClaimExtension() {
```

Comment:
I would use the imperative tense here, instead of past tense: `enableCoordinatorClaimExtension()` (without the d in enabled)

Reply:
Ah, definite typo here. Adjusted.